test/integration/main_test.go

```diff
@@ -184,12 +184,13 @@ func deleteAllPods(t *testing.T, namespace string) {
 }
 
 func checkPodsLogs(t *testing.T, kubeClient *kubernetes.Clientset, message string, newLogsOnly ...bool) {
+	timeNow := metav1.NewTime(time.Now())
+	timeNow.Add(-2*time.Second)
```

Comment:
If we use newLogsOnly=false, it could be safer then rely on (arbitrary) time before Now

Reply:
This would cause problems and probably break the tests even more.  If I use newLogsOnly=false, then it’s almost certain that that the check for `.*Wrote \d+ records to disk in \d+` will pass immediately, as it will be in logs already.. so tests of Logs and Event will get old archive again and fail. But I agree that it’s not the best solution now, I will try to find a different one.